source/_docs/automation/trigger.markdown

```diff
@@ -319,22 +319,27 @@ A very thorough explanation of this is available in the Wikipedia article about
 Fires when a [tag](/integrations/tag) is scanned. For example, a NFC tag is
 scanned using the Home Assistant Companion mobile application.
 
-{% raw %}
-
 ```yaml
 automation:
   trigger:
     platform: tag
     tag_id: A7-6B-90-5F
 ```
 
-{% endraw %}
+Or trigger on multiple possible tags:
```

Comment:
I sometimes wonder, should we have both examples or just have an example with multiple, from which one can derive it works with multiple?

Reply:
Reduced it with one example for multiple tags/devices.